python/test.py

```diff
@@ -189,6 +189,27 @@ def main():
 
         assert data.strip() == expected.strip()
 
+        print("Sending neighborhood request")
+        neighborhood = requests.get(
+                url + "/v0/pipelines/" + str(pipeline.pipeline_id) + "/egress/TRANSACTIONS_WITH_DEMOGRAPHICS?query=neighborhood&mode=snapshot&format=csv",
+                json = {'before': 10, 'after': 20, 'anchor': ['2008-01-03 05:38:14', 0.0, 'John', 'New York'] })
```

Comment:
yes, how about a test here with the wrong schema?

Reply:
Cleaner to use string interpolation here. I think this should work.

```
f'{url}/v0/pipelines/{pipeline.pipeline_id}/egress/TRANSACTIONS_WITH_DEMOGRAPHICS?query=neighborhood&mode=snapshot&format=csv',
```